### DISCUSSION TITLE
csv module?

### DISCUSSION THREAD

**Discussion by uvtc** [janet-lang/janet#663] [Q&A]
Is there a csv module available for Janet?

**Comment by uvtc** [comment]
I don't see it in [spork](https://github.com/janet-lang/spork) or in [Awesome Janet](https://github.com/ahungry/awesome-janet).

*Edit:* Forgot to check [pkgs](https://github.com/janet-lang/pkgs)! Didn't see one there either though.

**Comment by jcmkk3** [comment]
There is [janet-csv](https://github.com/zenlor/janet-csv). I haven't tried it, but it looks like it would probably work for very simple uses.

**Comment by uvtc** [comment]
Great! Thank you! Trying it out now.

**Reply by sogaiu** [reply]
May be obvious, but FWIW, there are some example invocations here: https://github.com/zenlor/janet-csv/tree/master/test

Also a small example of using it here: https://gitlab.com/sogaiu/janet-peg-samples/-/blob/master/samples/zenlor/janet-csv.janet
